js/ui/backgroundMenu.js

```diff
@@ -18,8 +19,20 @@ const BackgroundMenu = new Lang.Class({
 
         this.addSettingsAction(_("Change Background…"), 'gnome-background-panel.desktop');
         this.addMenuItem(new PopupMenu.PopupSeparatorMenuItem());
-        this.addSettingsAction(_("Display Settings"), 'gnome-display-panel.desktop');
-        this.addSettingsAction(_("Settings"), 'gnome-control-center.desktop');
+
+        this.addAction(_("Add App"), Lang.bind(this, function() {
```

Comment:
I don't think that's a good idea. `addSettingsAction()` is meant to be used for panels of gnome control center, and I don't think it fits well for opening something else like GS. Unless I'm missing something obvious I think that using `addAction()`, either with or without the app activation context, is probably the right thing to do.